rules/do-you-always-avoid-on-error-resume-next-vb-only/rule.md

```diff
@@ -1,17 +1,16 @@
 ---
 type: rule
-archivedreason: 
 title: Do you always avoid On Error Resume Next? (VB Only)
-guid: 44ff5d65-56bc-4b02-957d-378c8c0bfc5c
 uri: do-you-always-avoid-on-error-resume-next-vb-only
-created: 2013-09-11T22:01:01.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
 related: []
 redirects:
-- do-you-always-avoid-on-error-resume-next-(vb-only)
-
+  - do-you-always-avoid-on-error-resume-next-(vb-only)
+created: 2013-09-11T22:01:01.000Z
+archivedreason: Archiving as this rule isn't valid anymore
```

Comment:
I would explain that VB is not relevant anymore so we are archivingit

Reply:
Done